packages/preview/knowledge-key/1.0.0/lib.typ

```diff
@@ -0,0 +1,62 @@
+
+#let knowledge-key(
+  title: [Paper Title],
+
+  // An array of authors. For each author you can specify a name,
+  // department, organization, location, and email. Everything but
+  // but the name is optional.
+  authors: (),
+
+  // The article's paper size. Also affects the margins.
+  paper-size: "us-letter",
+
+  // The content.
+  body
+) = {
+  let line_skip = 0.4em
+  let font_size = 6pt
+  let level1_color = "#8c195f";
+  let level2_color = "#a12b66"; 
+  let level3_color = "#b63d6d"; 
+  let level4_color = "#cb4f74"; 
+  let level5_color = "#C63B65"; 
+
+  show: set block(below: line_skip)
+  show: set par(leading: line_skip, justify: true)
+  
+  // Configure the page.
+  set page(
+    paper: "a4",
```

Comment:
@ngoetti I think this should reflect [`paper-size`](https://github.com/typst/packages/pull/1020/files#diff-d9086d6821a58cdc5301a38b12c4809ff6e4d0295f875d8c79638110b8224430R11), no?

Reply:
The typst docu that I found calls it paper: https://typst.app/docs/reference/layout/page/#parameters-paper

![grafik](https://github.com/user-attachments/assets/3bf9e916-dbc1-405a-ba4c-16dc37672a1a)

Reply:
Sorry I mean — rather than hard-coding a4? Not sure if there's a way to override that with params?

Reply:
Ah I see, yes you are right. I have adjusted my repo to fix this: https://github.com/ngoetti/knowledge-key
Do you think I should create a new merge request so that this is fixed here aswell, or is the change to small to be worth it yet?

Reply:
I'm not sure what the typst team preferences are but I think it would be fine to make a new PR :) Thank you for the confirmation and fix!! :D

Reply:
It is really up to you, having a patch release to fix that bug is fine, waiting for more changes before releasing another version is okay too. :)